keras/backend/mxnet_backend.py

```diff
@@ -2049,7 +2051,7 @@ def concatenate(tensors, axis=-1):
     symbols = [t.symbol for t in tensors]
 
     if axis == 0 and py_all([is_sparse(t) for t in tensors]):
-        return KerasSymbol(mx.sym.sparse.concat(*symbols, dim=axis))
+        return KerasSymbol(mx.sym.sparse.concat(*symbols, dim=axis), stype='csr')
```

Comment:
similar to dot and mean ops, you don't need to do anything special now for concat op